Content.Server/Database/ServerDbBase.cs

```diff
@@ -228,7 +231,8 @@ private static HumanoidCharacterProfile ConvertProfiles(Profile profile)
                 jobs,
                 (PreferenceUnavailableMode) profile.PreferenceUnavailable,
                 antags.ToList(),
-                traits.ToList()
+                traits.ToList(),
+                loadouts.ToList()
             );
```

Comment:
As above this ToList looks unnecessary.

Reply:
`Argument type 'System.Collections.Generic.IEnumerable<string>' is not assignable to parameter type 'System.Collections.Generic.IReadOnlyList<string>'`